src/pages/login/login.module.css

```diff
@@ -0,0 +1,87 @@
+.loginPage {
```

Comment:
CSS properties should be `kebab-case`

Also I think this styling may be too much? I understand that you probably wanted to present the visual beauty of the component, but we have designs coming in and they will probably render your styling work obsolete 😅 

Like @jjleigh stressed in the beginning of the project, we are building UGLY first and beautifying later.